spec/Staking.md

```diff
@@ -351,6 +351,39 @@ share of "0xAB..CD" = Math.floor(35 * (1000 / (500 + 1000))) = 23
 remaing share (of author) = 110 - (11 + 23) = 76
 ```
 
+## ChangeParameters
+This transaction will change the common parameters when more than half of the stakeholders agree.
+It does not change other fields of the scheme file because there are fields related to the genesis block.
+
+It also does not provide a voting feature.
+The vote initiator should collect the signatures through the off-chain.
+
+This transaction increases the `seq` of `Metadata` and changes the `params` of `Metadata`.
+
+### Action
+`[ 0xFF, metadata_seq, new_parameters, ...signatures ]`
+
+#### metadata_seq
+The transaction fails if the metadata_seq is different from the `seq` of `Metadata` and is introduced to prevent replay attacks.
+
+#### new_parameters
+```
+new_parameters := [ new_parameter(, new_parameter)* ]
+new_parameter := [ key, value ]
```

Comment:
We use different conventions for the key of common parameters between rust(snake_case) and JSON (camelCase). What's the convention for the transaction?